app/models/office.rb

```diff
@@ -15,4 +15,10 @@ class Office < ActiveRecord::Base
   def managers
     users.where(office_id: id, role: 'manager')
   end
+
+  def business_entities
```

Comment:
The financial report shouldn't be an issue, as it loads applications and then reads back to the jurisdiction > business_entity.